ee/clickhouse/queries/test/__snapshots__/test_paths.ambr

```diff
@@ -1275,7 +1275,7 @@
                  WHERE team_id = 2
                    AND (event = '$pageview')
                    AND timestamp >= toDateTime('2012-01-01 00:00:00')
-                   AND timestamp <= toDateTime('2012-01-02 23:59:59')
```

Comment:
@timgl , looks like there's some differences between CH version 21.11 (on which snapshot tests ran on your PR) and 22.3 (which is the new default).

Just FYI. I'm not _yet_ sure how these changes could've happened, but will investigate if you don't end up getting to it